src/main/java/uk/gov/ons/ctp/response/casesvc/domain/model/Case.java

```diff
@@ -1,23 +1,23 @@
 package uk.gov.ons.ctp.response.casesvc.domain.model;
 
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.PostLoad;
+import jakarta.persistence.Table;
+import jakarta.persistence.Transient;
+import jakarta.persistence.Version;
```

Comment:
You will see many changes throughout for certain packages from `javax` to `jakarta`